modules/exploits/windows/http/git_lfs_rce.rb

```diff
@@ -22,10 +22,10 @@ def initialize(info = {})
         'Description' => %q{
           A critical vulnerability (CVE-2020-27955) in Git Large File Storage (Git LFS), an open source Git extension for
           versioning large files, allows attackers to achieve remote code execution if the Windows-using victim is tricked
-          into cloning the attacker’s malicious repository using a vulnerable Git version control tool
+          into cloning the attacker's malicious repository using a vulnerable Git version control tool
         },
         'Author' => [
-          'Dawid Golunski ', # Discovery
+          'Dawid Golunski', # Discovery
```

Comment:
This was to satisfy Rubocop, didn't want to open a PR to handle a single whitespace issue.